vpr/src/base/constraints_load.cpp

```diff
@@ -3,7 +3,7 @@
 static void print_region(FILE* fp, Region region);
```

Comment:
add to region class, (for each function, move to the class so it can print itself partition for partition, etc.)

Reply:
can be helper function in the class, or a global function after the class